docs/get-started.mdx

```diff
@@ -0,0 +1,304 @@
+---
+title: Get started
```

Comment:
@christopher-hakkaart, do you have any thoughts on these quick starts? Did I leave any obvious omissions that I should include with these? Thanks!

Reply:
For this page specifically? Or all pages in the PR?

Reply:
@christopher-hakkaart, oh, just this page. Thanks!